mobile/voice_journal_app/lib/home.dart

```diff
@@ -1,43 +1,71 @@
 import 'package:flutter/material.dart';
+import 'theme.dart';
 
 void main() {
   runApp(
     MaterialApp(
       home: Scaffold(
```

Comment:
To turn this into a class, we'll want to place everything in the `Scaffold` in the `build` function of the class, just like it is right now in the `MyHomePage` class in the `main.dart` file